### PR TITLE
Language improvements

### PR DESCRIPTION
**Pull request by discombobulateme**
- Suggested several small improvements in texts, making them smaller and avoiding dubious adjectives such as: good citizens and common sense.

- Added links to better explain some terminologies

- Suggested a new title from Projects to Open Sauce Projects to promote the project and differentiate our projects from the ones we support

### CODE REVIEW

**Review by christian-bromann — DISMISSED** [review]
Great work 👍

File: content/docs/contributing.md [hunk 1]
```diff
@@ -4,15 +4,17 @@ description: "Contributing to Open Source"
 draft: false
 ---
 
-Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. All Saucers are expected to follow common sense regarding the type of information and code is contributed to upstream open source projects. If you are in doubt, please reach out to the OSPO or ask your manager for direction.
+Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. Saucers are responsible to preserve our valuable information and code. OSPO and your manager can help you understand and determine what is open sourceable and what must remain private. In doubt, always ask.
 
 ### Summary
 
-We highly encourage contributions to existing open source projects. There are some requirements before you start patching, and guides to help you figure out the right path:
+We highly encourage contributions to [existing open source projects](https://opensource.saucelabs.com/projects/).
```
Comment (diemol) [inline content/docs/contributing.md]: The original phrase was meant to say that OSPO encourages contributions to any project that is open source, given that the guidelines are respected. 
The new phrase suggests contributing only to the open source projects featured in our OSS website.

File: content/docs/contributing.md [hunk 2]
```diff
@@ -4,15 +4,17 @@ description: "Contributing to Open Source"
...
 
-- Patches generally don't require any review from the Open Source Program Office
+There are some requirements before you start patching, and guides to help you figure out the right path:
+
+- [Patches](https://semver.org/) generally don't require any review from the Open Source Program Office
```
Comment (diemol) [inline content/docs/contributing.md]: The original meaning is not tied to `semver`, but to the general meaning of what a patch [is](https://en.wikipedia.org/wiki/Patch_(computing)).

**Review by diemol — CHANGES_REQUESTED** [review]
Thank you, @discombobulateme!

Just a couple of comments.

File: content/docs/contributing.md (hunk 1 above)
Comment (discombobulateme) [reply content/docs/contributing.md]: What do you think about this phrase? The idea behind this change was to easier the path between the wish and a project :)

We highly encourage contributions to any project that is open source, given that the guidelines are respected. Don't know where to start? Here you can find [Sauce Labs open source projects](https://opensource.saucelabs.com/projects/) and here you can find a list of [Open Source projects which support underrepresented groups](https://opensourcediversity.org/#projects) and where you can participate.

File: content/docs/contributing.md (hunk 2 above)
Comment (discombobulateme) [reply content/docs/contributing.md]: Makes super sense! I will change the link :) thank you for checking!

File: content/docs/contributing.md (hunk 1 above)
Comment (diemol) [reply content/docs/contributing.md]: I'd prefer more something along: "We highly encourage contributions to any open source project that aligns with the guidelines. If you are looking for projects to contribute, here are some suggestions: ...", and then a list of links of projects that we want to suggest.

**Comment by discombobulateme** [comment]
Just pushed the changes suggested by @diemol :)

**Review by diemol — APPROVED** [review]
Thank you!

**Pull request merged**